interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -142,14 +200,18 @@ public function getPatientdata($pid, $encounter)
             <mname>" . xmlEscape($result['mname']) . "</mname>
             <lname>" . xmlEscape($result['lname']) . "</lname>
             <suffix>" . xmlEscape($result['suffix']) . "</suffix>
+            " . $previous_names . "
             <birth_fname>" . xmlEscape($result['birth_fname']) . "</birth_fname>
             <birth_mname>" . xmlEscape($result['birth_mname']) . "</birth_mname>
             <birth_lname>" . xmlEscape($result['birth_lname']) . "</birth_lname>
-            <street>" . xmlEscape($result['street']) . "</street>
-            <city>" . xmlEscape($result['city']) . "</city>
-            <state>" . xmlEscape($result['state']) . "</state>
-            <postalCode>" . xmlEscape($result['postal_code']) . "</postalCode>
-            <country>" . xmlEscape($result['country_code']) . "</country>
+            <use>" . xmlEscape('HP') . "</use>
+            <street>" . xmlEscape($result['street'] ?? '') . "</street>
+            <street>" . xmlEscape($result['street_line_2'] ?? '') . "</street>
+            <city>" . xmlEscape($result['city'] ?? '') . "</city>
+            <state>" . xmlEscape($result['state'] ?? '') . "</state>
+            <postalCode>" . xmlEscape($result['postal_code'] ?? '') . "</postalCode>
+            <country>" . xmlEscape($result['country_code'] ?? '') . "</country>
+            " . $previous_addresses . "
```

Comment:
All of the previous address stuff looks great @sjpadgett.  Good job!

Reply:
Thanks, I'm working hard to get a good upcoming performance review from Brady and hopefully a long overdue raise!:)
Anyway, I love our newest human readable HTML CCDA. A very fine job Stephen.
It's almost looks like we know what we're doing!:)

Reply:
performance review 110%
yearly salary raise x2 :)